x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/expandable_section/expandable_section_results.tsx

```diff
@@ -0,0 +1,146 @@
+import { SavedSearchQuery } from '../../../../../contexts/ml';
+
+import { defaultSearchQuery, DataFrameAnalyticsConfig, SEARCH_SIZE } from '../../../../common';
+
+import { ExpandableSection, ExpandableSectionProps } from '../expandable_section';
+import { IndexPatternPrompt } from '../index_pattern_prompt';
+
+const showingDocs = i18n.translate(
+  'xpack.ml.dataframe.analytics.explorationResults.documentsShownHelpText',
+  {
+    defaultMessage: 'Showing documents for which predictions exist',
+  }
+);
+
+const showingFirstDocs = i18n.translate(
+  'xpack.ml.dataframe.analytics.explorationResults.firstDocumentsShownHelpText',
+  {
+    defaultMessage: 'Showing first {searchSize} documents for which predictions exist',
+    values: { searchSize: SEARCH_SIZE },
+  }
+);
+
+const getResultsSectionHeaderItems = (
+  columnsWithCharts: EuiDataGridColumn[],
+  tableItems: Array<Record<string, any>>,
+  rowCount: number,
+  colorRange?: ReturnType<typeof useColorRange>
+): ExpandableSectionProps['headerItems'] => {
+  return columnsWithCharts.length > 0 && tableItems.length > 0
+    ? [
+        {
+          id: 'explorationTableTotalDocs',
+          label: (
+            <FormattedMessage
+              id="xpack.ml.dataframe.analytics.exploration.explorationTableTotalDocsLabel"
+              defaultMessage="Total docs"
+            />
+          ),
+          value: rowCount,
+        },
+        ...(colorRange !== undefined
+          ? [
+              {
+                id: 'colorRangeLegend',
+                value: (
+                  <ColorRangeLegend
+                    colorRange={colorRange}
+                    title={i18n.translate(
+                      'xpack.ml.dataframe.analytics.exploration.colorRangeLegendTitle',
+                      {
+                        defaultMessage: 'Feature influence score',
+                      }
+                    )}
+                  />
+                ),
+              },
+            ]
+          : []),
+      ]
+    : 'loading';
+};
+
+interface ExpandableSectionResultsProps {
+  colorRange?: ReturnType<typeof useColorRange>;
+  indexData: UseIndexDataReturnType;
+  indexPattern?: IndexPattern;
+  jobConfig?: DataFrameAnalyticsConfig;
+  needsDestIndexPattern: boolean;
+  searchQuery: SavedSearchQuery;
+}
+
+export const ExpandableSectionResults: FC<ExpandableSectionResultsProps> = ({
+  colorRange,
+  indexData,
+  indexPattern,
+  jobConfig,
+  needsDestIndexPattern,
+  searchQuery,
+}) => {
+  const { columnsWithCharts, tableItems } = indexData;
+
+  // Results section header items and content
+  const resultsSectionHeaderItems = getResultsSectionHeaderItems(
+    columnsWithCharts,
+    tableItems,
+    indexData.rowCount,
+    colorRange
+  );
+  const resultsSectionContent = (
+    <>
+      {jobConfig !== undefined && needsDestIndexPattern && (
+        <IndexPatternPrompt destIndex={jobConfig.dest.index} />
```

Comment:
Looks like the error message here needs some padding on the left:

![image](https://user-images.githubusercontent.com/7405507/95088068-60d44400-071a-11eb-82d3-3236b2a4f036.png)

Reply:
Fixed padding in b1b57d0.